src/main/java/lotto/Service/OutputService.java

```diff
@@ -0,0 +1,49 @@
+package lotto.Service;
+
+import java.util.List;
+import lotto.Enum.LottoResult;
+import lotto.Lotto;
+
+public class OutputService {
+
+
+    private static final String LOTTO_OPENING_BRACKET = "[";
+    private static final String LOTTO_CLOSING_BRACKET = "]";
+    private static final String ERROR_MESSAGE = "[ERROR]";
+
+    public static void askStatmement(String askStatement) {
+        System.out.println(askStatement);
+    }
+
+    private static String getSingleLottoNumber(List<Integer> numbers, int idx) {
+        String ret = "";
+
+        ret = ret.concat(numbers.get(idx).toString());
+        if (idx != numbers.size() - 1) {
+            ret = ret.concat(", ");
+        }
+
+        return ret;
+    }
+
+    private static String makeLottoPrintStatement(List<Integer> sortedNumbers) {
+        String statement = LOTTO_OPENING_BRACKET;
+
+        for (int number = 0; number < sortedNumbers.size(); number++) {
+            statement = statement.concat(getSingleLottoNumber(sortedNumbers, number));
```

Comment:
배열 출력을 형식에 맞게 출력하기 위해 concat을 사용하여 구현하신 것 같아요.
```suggestion
int[] arr1 = {1,2,3,4,5};
System.out.println(Arrays.toString(arr1));  // [1, 2, 3, 4, 5]
List<Integer> arr2 = Arrays.asList(1,2,3,4,5);
System.out.println(arr2);  // [1, 2, 3, 4, 5]
```
식으로 출력이 가능합니다.